Adding link to full datadog.yaml file (#4456)
DataDog/documentation: content/en/agent/guide/agent-configuration-files.md

{{% /tab %}}
{{< /tabs >}}

> Note: [A full example of the `datadog.yaml` file can be seen in the `datadog-agent` Github repository][1].

Author: ```suggestion
Note: [A full example of the `datadog.yaml` file is available in the `datadog-agent` Github repository][1].
```
